index.md

```diff
@@ -17,55 +20,79 @@ run `pipeline.wdl` using
 java -jar cromwell-<version>.jar run -i inputs.json pipeline.wdl
 ```
 
-The inputs JSON can be generated using WOMtools as described in the [WOMtools
-documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/). Note that
-not some inputs should not be used! See [this page](inputs.md) for more
-information.
+### Inputs
+Inputs are provided through a JSON file. The primary inputs are described
+below, but additional inputs are available.
+A template containing all possible inputs can be generated using
+Womtool as described in the
+[WOMtool documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
+See [this page](inputs.md) for some additional general notes and information
+about pipeline inputs.
 
-The primary inputs are described below, additional inputs (such as precommands
-and JAR paths) are available. Please use the above mentioned WOMtools command
-to see all available inputs.
-
-| field | type | |
-|-|-|-|
-| sampleConfigFiles | `Array[File]` | The sample configuration files. See below for more details. |
-| refFasta | `File` | Reference fasta file. |
-| refFastaIndex | `File` | Index for the reference fasta files. |
-| refDict | `File` | The dict file for the referene fasta. |
-| dbsnpVCF | `File` | Reference dbSNP VCF file. |
-| dbsnpVCFindex | `File` | Index for the reference dbSNP VCF file. |
-| outputDir | `String` | The directory where the output will be placed. |
-| sample.library.readgroup.<br />mapping.bwaMem.referenceFasta | `File` | Reference fasta file from the BWA mem index. |
-| sample.library.readgroup.<br />mapping.bwaMem.indexFiles | `Array[File]` | The BWA mem index files assciated with the reference fasta. |
+The following are the bare minimum inputs required:
+```JSON
+{
+  "pipeline.bwaIndex": {
+    "fastaFile": "File: The fasta file from the bwa index",
```

Comment:
"File: The fasta file from the bwa index" -> absolute path to the fasta file that was used to build the bwa index

If absolute path is needed for the following variables, I would say that explicitly below too.